cmd/ko/commands.go

```diff
@@ -93,15 +94,27 @@ func addKubeCommands(topLevel *cobra.Command) {
 	addLocalArg(apply, lo)
 	addFileArg(apply, fo)
 	topLevel.AddCommand(apply)
+
 	resolve := &cobra.Command{
 		// TODO(mattmoor): Pick a better name.
 		Use:   "resolve -f FILENAME",
-		Short: `Print the input files with image references resolved to built/pushed image digests.`,
+		Short: "Print the input files with image references resolved to built/pushed image digests.",
 		Run: func(cmd *cobra.Command, args []string) {
 			resolveFilesToWriter(fo, lo, os.Stdout)
 		},
 	}
 	addLocalArg(resolve, lo)
 	addFileArg(resolve, fo)
 	topLevel.AddCommand(resolve)
+
+	publish := &cobra.Command{
+		Use:   "publish importpath",
```

Comment:
nit: capitalize IMPORTPATH for consistency with FILENAME above (take from kubectl)